src/FindNetworkEndpoint/index.ts

```diff
@@ -0,0 +1,39 @@
+import { INetwork } from './interface';
+
+export class FindNetworkEndpoint {
+  network: INetwork;
+  startNodeId: number;
+
+  constructor(startNodeId: number, fromIds: number[], toIds: number[]) {
+    this.network = {
```

Comment:
If the array are not the same length it should throw error here not in the find method